### PR TITLE
fix(configuration): correcting grammar

### PR DESCRIPTION
**Pull request by bhavya9107**

### CODE REVIEW

File: src/content/configuration/externals.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ contributors:
   - zefman
 ---
 
-The `externals` configuration option provides a way of excluding dependencies from the output bundles. Instead, the created bundle relies on that dependency to be present in the consumer's environment. This feature is typically most useful to __library developers__, however there are a variety of applications for it.
+The `externals` configuration option provides a way of excluding dependencies from the output bundles. Instead, the created bundle relies on that dependency to be present in the consumer's environment. This feature is typically most useful to __library developers__, however, there are a variety of applications for it.
```
Comment (byzyk) [inline src/content/configuration/externals.md]: I'm not sure whether it would make more sense if we would start a new sentence at `However` part.

Comment (bhavya9107) [reply src/content/configuration/externals.md]: I think, with a new line, it will make more sense. So, should I change it?

Comment (byzyk) [reply src/content/configuration/externals.md]: I mean I guess it's fine even the way it is now. Not a big deal. Thanks!

**Comment by bhavya9107** [comment]
@byzyk @montogeek @EugeneHlushko Anything left for me to be addressed.

File: src/content/configuration/externals.md (hunk 1 above)
Comment (bhavya9107) [reply src/content/configuration/externals.md]: @byzyk reverted

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**